doc/mainpage.dox

```diff
@@ -0,0 +1,66 @@
+// -*- lsst-c++ -*-
+
+namespace lsst { namespace meas { namespace extensions { namespace ngmix {
+/**
+@mainpage lsst::meas::extensions::ngmix;  plugins using NGMIX fitters
+This package provides wrappers around certain algorithms in Eric Sheldon's NGMIX package.
+The wrappers allow those algorithms to be accessed from meas_base plugins, and provides
+the standard capabilities for SingleFrameMeasurement, including:
+
+- Initializing the schema of the output SourceCatalog.
+- Providing a measure() method for measuring each input exposure.
+- Writing the measured result to the measRecord provided.
+- Handling errors or unexpected results by setting flags in measRecord.
+
+The NGMIX algorithms do multiple Gaussian fits and return a Gaussian Mixture (GMix).
```

Comment:
I'd consider moving this paragraph, describing NGMIX and Gaussian mixture nodellings up to just after the first sentence. Then start a new paragraph at "the wrappers...". This way you get to explain the unique qualities of the plugin early before covering the generic features of SingleFrameMeasurement plugins.